### DISCUSSION TITLE
录音后的文件名字怎么改呢？

### DISCUSSION THREAD

**Discussion by SkyPan19** [ZuodaoTech/everyone-can-use-english#821] [Q&A]
怎样将每次自己录音后的音频的文件名字自动变成数字，以便之后在文件管理器里更好地找出来，而不是每次录音保存后的音频文件的名字是各种没有规律的字母和数字？

**Comment by an-lee** [comment]
不能改，改了在 Enjoy 里就播放不了了。想要另保存可以在 Enjoy 里直接下载。
不建议直接修改 EnjoyLibrary 文件夹里的任何文件，除非你知道会产生什么后果。
